vultr/vultr.py

```diff
@@ -686,9 +686,9 @@ def server_reboot(self):
             found using the v1/server/list call.
         """
         params = {'SUBID': subid}
-        return self.request('/v1/server/reboot', params, 'POST')
+        return self.request('/v1/server/reboot', params, 'POST', json=False)
```

Comment:
Why default json=False here? Generally I prefer to speak JSON in all cases with API's unless there is a specific reason not to.